### ISSUE TITLE
Auto replace feature

### ISSUE THREAD

**Issue by DerryBear** [trezy/Turk#23]
A feature that allows me to input ¬ but replaces it with Ratsignal when input into chat box. This is also very helpful for signing into registered nicknames.

**Comment by trezy** [comment]
@DerryBear, can you clarify? Are you just talking about string replacements?

**Comment by DerryBear** [comment]
I mean I type ¬ and hit enter but it shows in channels as me saying Ratsignal.

**Comment by trezy** [comment]
Is ¬ supposed to be Return? I'm not sure what that character is supposed to represent.

**Comment by DerryBear** [comment]
Ok lets use the ! symbol. 

I type 
! 
And press enter to post to channel
But because I have set ! To be replaced by Ratsignal it would appear in the channel as

DerryBear[XB1]  Ratsignal

**Comment by trezy** [comment]
Yup, that's simple string replacement. I'll add it the the `next` milestone.
